app/models/User.coffee

```diff
@@ -85,6 +86,19 @@ module.exports = class User extends CocoModel
   isSessionless: ->
     Boolean((utils.getQueryVariable('dev', false) or me.isTeacher()) and utils.getQueryVariable('course', false) and not utils.getQueryVariable('course-instance'))
 
+  getClientCreatorPermissions: ->
+    clientID = @get('clientCreator')
+    if !clientID and /@codeninjas.com$/i.test(me.get('email'))
+      clientID = "57fff652b0783842003fed00"  # hard coded for code ninjas since a lot of their users do not have clientCreator set
```

Comment:
Can we move this codeninja hard-coded stuff to core/utils.coffee?

E.g. `utils.getApiClientIdFromEmail(@get('email'))`